src/merge.c

```diff
@@ -1711,8 +1732,13 @@ int git_merge__iterators(
 
 	git_vector_foreach(&changes, i, conflict) {
 		int resolved = 0;
+		int file_favor = opts.file_favor;
+
+		/* Check for merge options in .gitattributes */
+		if ((error = git_merge__lookup_file_favor(repo, conflict->our_entry.path, &file_favor) < 0))
```

Comment:
I traced through the git source code. It uses `ours` here:

https://github.com/git/git/blob/master/merge-recursive.c#L875